### PR TITLE
Regenerated autotool generated files

### PR DESCRIPTION
**Pull request by dhrupadb**
Ran `automake` and `autoconf`

### CODE REVIEW

**Review by jkohen — APPROVED** [review]
I assume you've tested this by now. Thanks for taking care of it!

File: configure [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69 for collectd 5.5.2.git.
+# Generated by GNU Autoconf 2.69 for collectd 5.5.2.1425.gcba3710.
```
Comment (igorpeshansky) [inline configure]: Let's try to make the patch as clean as possible, please.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

**Comment by dhrupadb** [comment]
PTAL

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Comment by igorpeshansky** [comment]
dhrupadb was at Google at the time.

**Pull request merged**